src/main.py

```diff
@@ -284,10 +284,8 @@ def _handle_result(self, result):
             self._play_assistant_response(result.response_audio)
         elif result.transcript:
             logger.warning('%r was not handled', result.transcript)
-            self.say(_("I don’t know how to answer that."))
```

Comment:
So this is the case when we get text, but we don't have an action for it. I think this feedback is important? Do you have an example where this happens where it shouldn't?